src/components/widget/index.js

```diff
@@ -9,8 +9,8 @@ export default React.createClass({
 	render() {
 		return (
 			<div className="react-boilerplate-widget">
-				<h3>Hello, World!</h3>
-				<p>I made this thing.</p>
+				<h3 ref="header">Hello, World!</h3>
+				<p ref="text">I made this thing.</p>
```

Comment:
I don't think that the boilerplate should suggest adjusting components to expose `refs`, instead we should use the test utils that are shipped with React or perhaps [Enzyme](https://github.com/airbnb/enzyme).